tests/security/apparmor/aa_autodep.pm

```diff
@@ -34,7 +34,8 @@ use utils;
 sub run {
     my ($self) = @_;
 
-    my $aa_tmp_prof = "/tmp/apparmor.d";
+    my $aa_tmp_prof   = "/tmp/apparmor.d";
```

Comment:
I am not sure why there is 2 blank here, it is the tidy result ?

Reply:
Yes, for tidy.